domain/src/main/assets/prototype_exploration.json

```diff
@@ -849,7 +849,7 @@
       },
       "content": {
         "content_id": "content",
-        "html": "<p>Sort the following in ascending order, putting equal items in the same position.</p>"
+        "html": "<p>Sort the following in descending order, putting equal items in the same position.</p>"
```

Comment:
I think that the first drag & drop state also says 'putting equal items in the same position' but grouping isn't enabled for it. Can you please fix the text accordingly?

Reply:
done